tests/functional/testSettings.js

```diff
@@ -63,6 +67,11 @@ describe('Settings', function() {
   });
 
   it('Checking the tips and tricks settings', async () => {
+    //tips and tricks feature is hidden when help sheets are enabled, so skip this test when help sheets are enabled
+    if (await vpn.isFeatureEnabled('helpSheets')) {
```

Comment:
I think it would be better to use `this.skip()` rather than returning directly, so the test doesn't report as passing. https://mochajs.org/#inclusive-tests (ctrl+f for "skip at runtime")

Reply:
did not know this existed! thanks!